src/install/memcached/find-data.mdx

```diff
@@ -0,0 +1,12 @@
+---
+componentType: default
+headingText: Find and use data
+---
+
+Metrics are attached to these event types:
```

Comment:
This def needs more context. i know what this means, but let's say you're a new user to New Relic. Where do i go to see this relationship? You might want to point them to the data explorer tool via pathing, and give a high level breakdown of how to actually find their data (data explorer) and use it (query your data).

afaik all of these actions are housed in the Query your data component nowadays